.github/workflows/rust.yml

```diff
@@ -5,15 +5,31 @@ on:
       - .github/workflows/rust.yml
       - Cargo.toml
       - ci/**
-      - crates/**
+      - ibc/**
+      - ibc-core/**
+      - ibc-apps/**
+      - ibc-data-types/**
+      - ibc-clients/**
+      - ibc-primitives/**
+      - ibc-query/**
+      - ibc-testkit/**
+      - ibc-derive/**
```

Comment:
I think you can do `ibc*/**`, unless you want to be selective.

Reply:
yea, still being able to selectively comment out each crates works better for the development and debugging purposes imo.